doc/changelog/07-vernac-commands-and-options/14699-alizter+primitive-attribute.rst

```diff
@@ -0,0 +1,6 @@
+- **Added:**
+  :attr:`projections(primitive)` attribute for marking a record use
+  primitive projections.
```

Comment:
```suggestion
  :attr:`projections(primitive)` attribute for marking a record using
  primitive projections
```

Trivial picky detail: The period for the last sentence in these (you have only one) is after the `by Ali Caglayan).`, do you don't need one after `projections`

Reply:
Ah OK, this was confusing me since I saw some full stops in the change log elsewhere. In the make-changelog script perhaps we should write
```
Bla Bla Bla (don't end with a period)
```
or something.

Reply:
Yes, please, feel free to open a PR improving this. I've fixed many of these full stops before parentheses but at some point I've got tired and stopped caring about it.

Reply:
This comment was marked as resolved but I don't see the change to the changelog entry.

BTW, I would have fixed "marking a record use primitive projections" differently from Jim's suggestion: I would have suggested:
```suggestion
  :attr:`projections(primitive)` attribute to make a record use
  primitive projections
```

Reply:
Done.